packages/admin-ui-docs/docs/components/checkbox.mdx

```diff
@@ -511,31 +532,161 @@ function Example() {
+<li>Always add (Recommended) to the preselected option’s label.</li>
+  </ul>
+</p>
+</details>
+
+</PageSection>
+
+<PageSection
+  id={tabToc.designReference[3].id}
+  title={tabToc.designReference[3].value}
+>
+
+<details><summary>What should be the label of a Checkbox?</summary>
+<p>
+  <ul>
+    <li>Write the label so it corresponds to the value of the Checkbox.</li>
+    <li>Use sentence case but capitalize proper nouns.</li>
+  </ul>
+  <br /><br />
+  What to avoid:
+  <ul>
+    <li>Don't write long Checkbox labels. Use the help text or the Checkbox Group label to provide additional context when necessary.</li>
+    <li>Don't write labels that vary a lot in length between options.</li>
+    <li>Don't use personal pronouns. For example, write <em>Receive promotional emails</em> instead of <em>I want to receive emails</em>.</li>
+    <li>Don't include redundant words. For example, use only <em>S</em>, <em>M</em>, and <em>L </em>instead of <em>Size S</em>, <em>Size M</em>, and <em>Size L</em>.</li>
+    <li>Don't include negative words. For example, write <em>Receive promotional emails</em> instead of <em>I don't want to receive emails</em>.</li>
+  </ul>
+</p>
+</details>
+
+<details><summary>What should be the label of a Checkbox Group?</summary>
+<p>
+  <ul>
+    <li>Include the name of the entity that is being listed. For example, use the label <em>Fruits </em>when the options are <em>Apple</em>, <em>Orange </em>and <em>Banana</em>.</li>
+    <li>Visually hide the label of the Checkbox Group when the form section title already contains the name of the entity that is being listed.</li>
+    <li>Use sentence case, but capitalize proper nouns.</li>
+  </ul>
+  <br /><br />
+  What to avoid:
+  <ul>
+    <li>Don't include redundant words. For example, don't use <em>Product characteristics</em> as the label for a Checkbox Group inside a form page that includes <em>Product</em> in its title – use only <em>Characteristics</em>.</li>
+    <li>Don't use verbs. For example, instead of <em>Select characteristics</em>, use only <em>Characteristics</em>.</li>
+    <li>Don't use interrogations.</li>
+  </ul>
+</p>
+</details>
+
+<details><summary>What should be the help text of a Checkbox?</summary>
+<p>
+  <ul>
+    <li>Include a help text only for critical choices where the consequences of checking the option need to be explained in more detail.</li>
+    <li>Write the help text as if you are completing the following sentence: <em>If you check this option, you or the system will [help text]</em>.</li>
+    <li>Start with verbs in the simple present tense. For example, for the value <em>Fragile</em> the help text should be<em> Prints this information on the shipping label</em> and not <em>Print this information on the shipping label</em>.</li>
+    <li>Use sentence case, but capitalize proper nouns.</li>
+  </ul>
+  <br /><br />
+  What to avoid:
```

Comment:
<li>Don't include links. Use an alert component if this is necessary.</li>
    <li>Don't write more than one sentence.</li>
    <li>Don't use periods or commas.</li>